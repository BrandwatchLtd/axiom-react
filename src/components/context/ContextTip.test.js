import React from 'react';
import renderer from 'react-test-renderer';
import { ContextTip, Paragraph } from 'bw-axiom';

const getComponent = (props = {}) =>
  renderer.create(
    <ContextTip { ...props }>
      <Paragraph>Lorem ipsum</Paragraph>
    </ContextTip>
  );

describe('ContextTip', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
