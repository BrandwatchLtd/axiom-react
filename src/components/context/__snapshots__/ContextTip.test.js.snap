// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContextTip renders with defaultProps 1`] = `
<div
  className="ax-context-box ax-context-box--padding-small"
  style={
    Object {
      "maxHeight": undefined,
    }
  }
>
  <p
    className="ax-space--x2"
  >
    Lorem ipsum
  </p>
</div>
`;
